parsing/pprintast.ml

```diff
@@ -1626,9 +1640,12 @@ and structure_item ctxt f x =
         (value_description ctxt) vd
         (item_attributes ctxt) vd.pval_attributes
   | Pstr_include incl ->
-      pp f "@[<hov2>include@ %a@]%a"
+      (* Print "include functor" rather than attribute *)
+      let attrs, incl_fun = check_include_functor_attr incl.pincl_attributes in
+      pp f "@[<hov2>include%a@ %a@]%a"
```

Comment:
This would print `include` (and not `include functor`) since we do not consult `pincl_kind`